tools/clang/test/HLSLFileCheck/dxil/debug/noops_no_fold.hlsl

```diff
@@ -36,3 +36,5 @@ float4 main() : SV_Target {
   // CHECK: call void @llvm.donothing()
```

Comment:
This test is still checking for `@llvm.donothing()`.  The reason this doesn't fail is because dxilver has been set to 1.7, which is not yet available at all.  We should update this test and use SM 6.0, removing the `%dxilver 1.7` part.